### ISSUE TITLE
split handlers into separate files

### ISSUE THREAD

**Issue by JMurphyWeb** [emfoundation/ce100-app#146]
Use make-changes as a guide to split the handlers into separate files.

**Comment by nelsonic** [comment]
We have all _agreed_ that splitting out the `server/handlers.js` file into `server/handlers/my-handler.js` is a good idea for sanity and reducing merge conflicts as the `handlers.js` (_mega file_) would be changed on _every_ PR and thus create a headache...

**Comment by JMurphyWeb** [comment]
In order to make this a seamless merge, I will not change the handlers at all, only split them and then PR them. I don't think that this needs to be done before the open and merge-able PRs get reviewed/merged.

@nelsonic due to the way the PRs are chained off of each other,  I am pretty sure there will be no merge conflicts. I am about to start work on this issue now though, so if you would prefer to wait that is fine, i will pr and notify you when done.

**Comment by nelsonic** [comment]
@JMurphyWeb creating _new_ handlers inside the `handlers.js` ...?
![cat-sad](https://cloud.githubusercontent.com/assets/194400/18509642/83641c1e-7a71-11e6-92a5-956fe23f7c41.jpg)

And when we are _editing_ a _existing_ handler it's the _perfect_ opportunity to do it _right_ (i.e. split it out and thus make `handlers.js` smaller_)
![worth-doing](https://cloud.githubusercontent.com/assets/194400/18509704/ebae0c6c-7a71-11e6-811a-af8624c02f39.png)
